src/client-functions/client-function-builder.js

```diff
@@ -12,7 +12,7 @@ import getCallsite from '../errors/get-callsite';
 const DEFAULT_EXECUTION_CALLSITE_NAME = '__$$clientFunction$$';
 
 export default class ClientFunctionBuilder {
-    constructor (fn, options, callsiteNames = {}) {
+    constructor (fn, options, callsiteNames = {}, clientFunctionWrongTypeMessage = MESSAGE.clientFunctionCodeIsNotAFunction) {
```

Comment:
the existing error states that something
`is expected to be specified as a function`
but the handler is actually
`expected to be specified as a regular or client function`